tests/scripts/anisotropic_downsampling.sh

```diff
@@ -25,3 +26,21 @@ docker run \
 [ -d testoutput/tiff/color/4-4-1 ]
 [ $(find testoutput/tiff/color/2-2-1 -mindepth 3 -name "*.wkw" | wc -l) -eq 1 ]
 [ $(find testoutput/tiff/color/4-4-1 -mindepth 3 -name "*.wkw" | wc -l) -eq 1 ]
+
+docker run \
+  -v "${PWD}/testoutput:/testoutput" \
+  --rm \
+  scalableminds/webknossos-cuber:${CIRCLE_BUILD_NUM} \
+  wkcuber.downsampling \
+  --jobs 2 \
+  --from 4-4-1 \
+  --max 16 \
+  --buffer_cube_size 128 \
+  --layer_name color \
+  --scale 11.24,11.24,25 \
+  /testoutput/tiff
+[ -d testoutput/tiff/color/2-2-1 ]
+[ -d testoutput/tiff/color/4-4-1 ]
```

Comment:
Shouldn't you assert that `8-8-1` was created?

Reply:
True. I forgot to change that but it needs to be `8-8-2` and `16-16-4`